modules/gitops-release-notes-1-14-1.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-14.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-14-1_{context}"]
+= Release notes for {gitops-title} 1.14.1
+
+{gitops-title} 1.14.1 is now available on {OCP} 4.13, 4.14, 4.15, 4.16, and 4.17.
+
+[id="errata-updates-1-14.1_{context}"]
+== Errata updates
+
+[id="RHEA-XXXX:NNNN-gitops-1-14-1-security-update-advisory_{context}"]
+=== RHEA-XXXX:NNNN - {gitops-title} 1.14.1 security update advisory
+
+Issued: 2024-10-29
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHEA-XXXX:NNNN[RHEA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-14-1_{context}"]
+== Fixed issues
+
+* Before this update, resource limits were set on the operator container as a recommended practice. However, these limits caused functional issues on clusters with a large number of secrets and configmaps because the operator required more memory than allowed by these limits. With this update, the resource limits set on the manager container have been removed to minimize the impact on functionality. Efforts are focused to optimize memory consumption for future releases. link:https://issues.redhat.com/browse/GITOPS-5665[GITOPS-5665]
```

Comment:
```suggestion
* Before this update, resource limits were set on the operator container as a recommended practice. However, these limits caused functional issues on clusters with a large number of secrets and config maps because the operator required more memory than allowed by these limits. With this update, the resource limits set on the manager container have been removed to minimize the impact on functionality. Efforts are focused to optimize memory consumption for future releases. link:https://issues.redhat.com/browse/GITOPS-5665[GITOPS-5665]
```

Reply:
Fixed.